packages/currency/src/native.ts

```diff
@@ -75,6 +75,12 @@ export const nativeCurrencies: Record<NativeCurrencyType, (NativeCurrency & { na
       name: 'Near Testnet',
       network: 'aurora-testnet',
     },
+    {
+        symbol: 'ARETH',
+        decimals: 18,
+        name: 'Arbitrum Testnet',
+        network: 'arbitrumtestnet'
```

Comment:
Adding a dash seems to increase its readability (`arbitrum-testnet`). Is it possible?

Reply:
Done !

Reply:
I don't see it done here, but it's done in all other places.

Reply:
and indentation should be 2 spaces and not 4 as in the other blocks.